### PR TITLE
i18n(es): Translated `kontent-ai.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `kontent-ai.mdx` to spanish 🇪🇸

### CODE REVIEW

File: src/content/docs/es/guides/cms/kontent-ai.mdx [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+Esta implementación crea un nuevo objeto `DeliveryClient` usando credenciales del archivo `.env`.
+
+:::note[previsualizaciones]
+La `previewApiKey` es opcional. Cuando se usa, puedes [configurar cada consulta](https://github.com/kontent-ai/delivery-sdk-js#enable-preview-mode-per-query) al endpoint de la API de entrega para devolver las últimas versiones de los elementos de contenido independientemente de su estado en el flujo de trabajo. De lo contrario, solo se devuelven los elementos publicados.
+:::
+
+Finalmente, el directorio raíz de tu proyecto de Astro ahora debería incluir estos nuevos archivos:
+
+<FileTree title="Project Structure">
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/kontent-ai.mdx:101]:
```suggestion
<FileTree title="Estructura del proyecto">
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

**Pull request merged**